rust_src/remacs-sys/lib.rs

```diff
@@ -765,14 +765,14 @@ extern "C" {
 
 /// Lisp_Char_Table
 #[repr(C)]
-#[allow(dead_code)]
-enum ChartabSize {
+pub enum ChartabSize {
     Bits0 = 6,
     Bits1 = 4,
     Bits2 = 5,
     Bits3 = 7,
 }
 
+/// Lisp_Char_Table
```

Comment:
I'm not sure this comment adds anything :)

Reply:
Following the existing style...